buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -229,6 +229,31 @@ RUN mkdir /home/${USER}/openjdk_cache \
   && git remote add omr https://github.com/eclipse/openj9-omr.git \
   && git fetch --all
 
+# Install OpenSSL v1.1.0
+RUN cd /tmp \
+  && wget https://www.openssl.org/source/old/1.1.0/openssl-1.1.0g.tar.gz \
+  && tar -xzf openssl-1.1.0g.tar.gz \
+  && rm -f openssl-1.1.0g.tar.gz \
+  && cd /tmp/openssl-1.1.0g \
+  && ./config --prefix=/usr/local/openssl-1.1.0 \
```

Comment:
I'm not sure if the prefix should have openssl in it?

Reply:
When the prefix is `/usr/local/openssl-1.1.0` it gets installed so the executable is `/usr/local/openssl-1.1.0/bin/openssl`, and when the prefix is `/usr/local` the executable is `/usr/local/bin/openssl`. Which is preferred?

Reply:
I think the latter. The former does not put it on the PATH

Reply:
I tested this manually in the previous version of the container. Having issues finding the openssl lib.
```
/usr/local/openssl-1.1.0/bin/openssl: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```
Found this https://github.com/openssl/openssl/issues/4833, particularly https://github.com/openssl/openssl/issues/4833#issuecomment-348767030 which suggests
```
./config -Wl,-rpath,/usr/local/lib64
```
Note: `--prefix=/usr/local` is [default](https://github.com/openssl/openssl/blob/master/INSTALL#L105), so not needed

This seemed to solve the issue
```
# openssl version
OpenSSL 1.1.0g  2 Nov 2017
```